src/api/transactionWatch_unstable_submitAndWatch.md

```diff
@@ -127,7 +124,7 @@ No more event will be generated about this transaction.
 
 The `invalid` event indicates that the runtime has marked the transaction as invalid.
 
-This can happen for a variety of reasons specific to the chain, such as a bad signature, bad nonce, not enough balance for fees, invalid decoded transaction bytes etc.
+This can happen for a variety of reasons specific to the chain, such as a bad signature, bad nonce, not enough balance for fees, etc.
```

Comment:
Wanted to double check this with you: 
- in the error section we mention "no error is produced if the bytes of the `transaction`, once decoded, are invalid. Instead, an `invalid` notification will be generated"

If my understanding is correct, wrong inputs (non-hex strings) should lead to an error. Failing to decode the transaction bytes to the shape of what the chain expects should return an `Invalid` event. Thanks 🙏

Reply:
This sentence is not about non-hex string. If for example you send `0xaabbccdd` for the transaction, you'll get an `invalid` event rather than a JSON-RPC error.